tests/skimage/measure/test_regionprops.py

```diff
@@ -1511,6 +1512,16 @@ def test_extra_properties_table():
     assert out["bbox_list"][1] == [1] * 1
 
 
+def is_macos_intel_accelerate():
```

Comment:
So if I understand correctly, depending on the architecture the memory layout is different (non-contiguous) which will impact the results? 

And by explicitly copying we avoid non-contiguity. If so, why then does the test below need relaxing `atol` on MacOS intel accelerate?

Reply:
And you're right - the `copy` was a different fix - I accidentally left it in - have now removed it.